package.json

```diff
@@ -6,9 +6,10 @@
     "@lexical/history": "0.16.1",
     "@lexical/react": "0.16.1",
     "@mediapipe/tasks-vision": "0.10.14",
+    "@wireapp/api-client": "^27.1.0",
```

Comment:
don't think api-client should shoe up here, isn't it handled by core?

Reply:
That is true. Gonna remove it